sources/tools/Stride.Importer.Assimp/Material/Flags.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation and Contributors (https://dotnetfoundation.org/ & https://stride3d.net) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System;
+
+namespace Stride.Importer.Assimp.Material
+{
+    /// <summary>
+    /// Enumeration of the new Assimp's flags.
```

Comment:
A better explanation of this types would be nice.
It's ok however if you are not very familiar with graphics and/or meshes/Assimp. That is something that can be later expanded in a PR to improve documentation I think.